app/Services/MentionsService.php

```diff
@@ -701,17 +701,14 @@ protected function mapAttributes()
         $this->text = preg_replace_callback('`\{attribute:(.*?)\}`i', function ($matches) {
             $id = (int) $matches[1];
             $attribute = $this->attribute($id);
-            $fallback = '';
-            if (str_contains($matches[1], '|')) {
-                $fallback = substr($matches[1], strpos($matches[1], "|") + 1);
-            }
+            $fallback = Str::after($matches[1], '|');
```

Comment:
If a mention has no fallback and can't be viewed because of permission, it now shows "attribute:123" instead of _unknown_.